### PR TITLE
457 persons pages

### PR DESCRIPTION
**Pull request by joncppl**

### CODE REVIEW

File: dispatch/static/manager/src/js/components/Header.js [hunk 1]
```diff
@@ -40,7 +40,7 @@ export default function Header(props) {
                   <Link to='/sections/'><span className='pt-icon-standard pt-icon-properties' /> sections</Link>
                 </li>
                 <li className='c-header__dropdown__link'>
-                  <Link to='/people/'><span className='pt-icon-standard pt-icon-person' /> people</Link>
+                  <Link to='/persons/'><span className='pt-icon-standard pt-icon-person' /> people</Link>
```
Comment (psiemens) [inline dispatch/static/manager/src/js/components/Header.js]: Change the link text to "persons" as well

File: dispatch/static/manager/src/js/components/PersonEditor/PersonForm.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      displayImg: null
+    }
+  }
+
+  onDrop(files) {
+    this.props.update('image', files[0])
```
Comment (psiemens) [inline dispatch/static/manager/src/js/components/PersonEditor/PersonForm.js:20]: Does `Dropzone` have a setting to only allow single file uploads?

File: dispatch/static/manager/src/js/components/PersonEditor/PersonForm.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+          activeClassName='c-person-form__image__dropzone--active'>
+          <div
+            className='c-person-form__images__container'>
+            {this.state.displayImg || this.props.listItem.image ? null :
+              <div className='c-person-form__image__dropzone__text'>
+                Drop Image Here
+              </div>}
+            <img
+              className='c-person-form__images'
+              ref={(node) => { this.images = node }}
```
Comment (psiemens) [inline dispatch/static/manager/src/js/components/PersonEditor/PersonForm.js]: Is `this.images` used anywhere?

File: dispatch/static/manager/src/js/components/PersonEditor/PersonForm.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+            value={this.props.listItem.full_name || ''}
+            fill={true}
+            onChange={ e => this.props.update('full_name', e.target.value) } />
+        </FormInput>
+        <FormInput
+          label='Slug'
+          padded={false}
+          error={this.props.errors ? this.props.errors.slug : null}>
+          <TextInput
+            placeholder='Full Name'
```
Comment (psiemens) [inline dispatch/static/manager/src/js/components/PersonEditor/PersonForm.js]: Change placeholder to "Slug"

File: dispatch/static/manager/src/js/components/PersonEditor/PersonForm.js [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+            <img
+              className='c-person-form__images'
+              ref={(node) => { this.images = node }}
+              src={this.state.displayImg || this.props.listItem.image}/>
+          </div>
+        </Dropzone>
+
+        <div className='c-person-form__image__button'>
+          <AnchorButton
+            intent={Intent.SUCCESS}
```
Comment (psiemens) [inline dispatch/static/manager/src/js/components/PersonEditor/PersonForm.js]: Remove the `intent` (I think the default button is fine)

File: dispatch/static/manager/src/js/components/PersonEditor/PersonForm.js [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+          <TextInput
+            placeholder='Full Name'
+            value={this.props.listItem.full_name || ''}
+            fill={true}
+            onChange={ e => this.props.update('full_name', e.target.value) } />
+        </FormInput>
+        <FormInput
+          label='Slug'
+          padded={false}
+          error={this.props.errors ? this.props.errors.slug : null}>
```
Comment (psiemens) [inline dispatch/static/manager/src/js/components/PersonEditor/PersonForm.js]: It doesn't look like errors are working correctly for any of the fields

**Review by psiemens — CHANGES_REQUESTED** [review]
In addition to my code comments, the only thing that's missing is the toast messages in `ToasterReducer.js`

File: dispatch/static/manager/src/js/components/PersonEditor/PersonForm.js (hunk 6 above)
Comment (joncppl) [reply dispatch/static/manager/src/js/components/PersonEditor/PersonForm.js]: for persons, the model allows blank for all the fields, so the server doesn't return errors.
should that be changed? or are there other error conditions?

**Pull request merged**